download/alpha.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: base
+title: Alpha Download
+navgroup: download
+redirect_from:
+  - /alpha/
+  - /Alpha/
+---
+<div class="row">
+  <div class="small-12 columns">
+    <h1>Download</h1>
+
+    <h5>The alpha MonoDevelop release is: <strong>{{ site.data.alpharelease.version }}</strong></h5>
+    <p>Please choose your operating system to view the available packages. Source code is available on <a href="https://github.com/mono/monodevelop">GitHub</a> (viewing only) or as a <a href="https://download.mono-project.com/sources/monodevelop/">Tarball</a>.</p>
+
+    <dl id="mono-download" class="tabs horizontal" data-tab data-options="deep_linking: true; scroll_to_content: false">
+      <dd class="active"><a href="#download-mac"><i class="fa fa-apple"></i>&nbsp;Mac OS X</a></dd>
+      <dd><a href="#download-lin"><i class="fa fa-linux"></i>&nbsp;Linux</a></dd>
+      <dd><a href="#download-win"><i class="fa fa-windows"></i>&nbsp;Windows</a></dd>
+    </dl>
+
+    <div class="tabs-content" id="mono-download-panel">
+      <div class="panel content active" id="download-mac" style="padding: 20px 10px 10px 10px">
+        <h3>Visual Studio for Mac is available as a Mac Package (.pkg)</h3>
+        <div>
+          <a href="{{ site.data.alpharelease.monodevelop_mac_url }}" class="button radius"><i class="fa fa-download"></i> Download Visual Studio for Mac</a>
+          <p>Supported on Mac OS X 10.7 and later.</p>
```

Comment:
We might want to double check with the VSMac folks on this.

Reply:
I think it is 10.11. That version is in the Info.plist and is specified on the [system requirements web page](https://www.visualstudio.com/en-us/productinfo/vs2017-system-requirements-mac)